### PR TITLE
[ALLUXIO-2082] Allow s3a credentials to be stored in site properties

### PR DESCRIPTION
**Pull request by calvinjia**
https://alluxio.atlassian.net/browse/ALLUXIO-2082

### CODE REVIEW

File: core/common/src/main/java/alluxio/Constants.java [hunk 1]
```diff
@@ -417,6 +417,9 @@
   public static final String S3_ACCESS_KEY = "fs.s3n.awsAccessKeyId";
```
Comment (ifcharming) [inline core/common/src/main/java/alluxio/Constants.java]: shall we rename `S3_ACCESS_KEY` to `S3N_ACCESS_KEY` then?

File: core/common/src/main/java/alluxio/Constants.java [hunk 2]
```diff
@@ -417,6 +417,9 @@
...
   public static final String S3_SECRET_KEY = "fs.s3n.awsSecretAccessKey";
 
+  public static final String S3A_ACCESS_KEY = "aws.accessKeyId";
```
Comment (ifcharming) [inline core/common/src/main/java/alluxio/Constants.java]: the constant key and name do not match, can you add a comment to explain why?

**Comment by ifcharming** [comment]
thanks @calvinjia , I've left a couple questions.

File: core/common/src/main/java/alluxio/Constants.java (hunk 1 above)
Comment (calvinjia) [inline core/common/src/main/java/alluxio/Constants.java]: Good call, I just realized this is not public api.

File: core/common/src/main/java/alluxio/Constants.java (hunk 2 above)
Comment (calvinjia) [inline core/common/src/main/java/alluxio/Constants.java]: Updated, thanks.

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**